knowledge-base/kb-window-close-radwindow-dialog-in-mvvm.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How to close RadWindow dialog in MVVM using custom button
+description: Closing a RadWindow dialog from within the command that is being executed from a custom button specified in the template.
+type: how-to
+page_title: Using button command to close a RadWindow dialog in MVVM.
+slug: kb-window-close-radwindow-dialog-in-mvvm
+position: 0
+tags: window,close, mvvm, button, custom, template
+ticketid: 313798
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2019.1.116</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>RadWindow for WPF</td>
+	</tr>
+</table>
+
+## Description
+
+How to close a RadWindow Confirm dialog in MVVM scenario using custom button.
+
+## Solution
+
+* Create a custom style which targets RadConfirm (for example) dialog. 
+* Override the ControlTemplate by specifying your own one. You can add your button. 
+* Bind the command property of the button to your custom command using RelativeSource binding. 
+* To close the dialog in the executed method we need to pass the RadWindow to the CommandParameter using the same approach with RelativeSource binding. 
```

Comment:
I'd change the "we" to "you" in the last sentence for consistency, Also, consider a numbered list since these are consecutive steps.